lib/osm_community_index/local_chapter.rb

```diff
@@ -0,0 +1,59 @@
+module OsmCommunityIndex
+  class LocalChapter
+    attr_reader :id, :url
+
+    def initialize(id, url)
+      @id = id
+      @url = url
+    end
+
+    def self.local_chapters
+      @chapters = load_local_chapters
+    end
+
+    def self.load_local_chapters
+      community_index = OsmCommunityIndex.community_index
+      local_chapters = []
+      community_index["resources"].each do |id, resource|
+        resource.each do |key, value|
+          next unless key == "type" && value == "osm-lc" && id != "OSMF"
+
+          # name comes via I18n
+          url = resource["strings"]["url"]
+          local_chapters.push(LocalChapter.new(id, url))
+        end
+      end
+      local_chapters
+    end
+
+    def self.add_to_i18n
+      community_index = OsmCommunityIndex.community_index
+      files = Dir.children(Rails.root.join("node_modules/osm-community-index/i18n/"))
```

Comment:
`Dir.glob(Rails.root.join("node....i18n/*")` (note the *) will return an array of file paths, so then you can skip the path rebuilding bit two lines further down.